src/content/configuration/output.mdx

```diff
@@ -172,6 +172,8 @@ module.exports = {
 
 The format of chunks (formats included by default are `'array-push'` (web/WebWorker), `'commonjs'` (node.js), `'module'` (ESM), but others might be added by plugins).
 
+T> The default value of this option is depending on the [`target`](/configuration/target/) and [`output.module`](#outputmodule) setting. For more details, search for `"workerChunkLoading"`: [in the webpack defaults](https://github.com/webpack/webpack/blob/master/lib/config/defaults.js).
```

Comment:
You meant searching `chunkFormat` here?

Reply:
oops. i copied over the tip from `workerChunkLoading`